app/assets/stylesheets/govuk-component/_title.scss

```diff
@@ -1,16 +1,18 @@
-.govuk-title {
+// scss-lint:disable SelectorFormat
+.pub-c-title {
   @include responsive-vertical-margins;
 
-  .context {
-    @include core-24;
-    color: $secondary-text-colour;
+  &--long .pub-c-title__text {
```

Comment:
Seems as if here, since the long modifier only impacts the text element, we could have a modifier class on `.pub-c-title_text` instead?